index.html

```diff
@@ -12,6 +12,9 @@
     <div class="container">
       <div class="container-book">
         <div class="container-img">
+          <div class="container-badge">
+            <img class="img" src="./assets/badge.png" alt="badge of detail">
```

Comment:
You can use `.container-badge` styles on this img and remove the div. Let's try to keep the DOM as simple as possible